hystrix/README.md

```diff
@@ -74,7 +74,7 @@ GitHub fallback = (owner, repo) -> {
   }
 };
 
-GitHub github = HystrixFeign.builder()
-                            ...
-                            .target(GitHub.class, "https://api.github.com", fallback);
+HystrixFeign.Builder builder = HystrixFeign.builder();
+... 
+GitHub github = builder.target(GitHub.class, "https://api.github.com", fallback);
```

Comment:
Because the change is in a terminal method you need it's original type :disappointed:. One option is to move create a `fallback()` method which could retain the fluent style. Another is to create an `AbstractBuilder<B>` that returns `B` from each method allowing downstream builders to have custom terminal methods.